docs_2.0/manual_EN/4.ngql-guide/12.vertex-statements/1.insert-vertex.md

```diff
@@ -0,0 +1,81 @@
+# INSERT VERTEX
+
+```ngql
+INSERT VERTEX <tag_name> (prop_name_list) [, <tag_name> (prop_name_list), ...]
+     {VALUES | VALUE} VID: (prop_value_list[, prop_value_list])
+
+prop_name_list:
+  [prop_name [, prop_name] ...]
+
+prop_value_list:
+  [prop_value [, prop_value] ...]
+```
+
+The `INSERT VERTEX` statement inserts a vertex or vertices into a graph space in a Nebula Graph instance.
+
+* `tag_name` denotes the `tag` (vertex type), which must be created before `INSERT VERTEX`.
+* `prop_name_list` is the property name list in the given `tag_name`.
+* `VID` is the vertex ID. In Nebula Graph 2.0.0-Alpha, only string type VID is supported. The maximum VID length is set when creating the graph space. For detail information on the maximum VID length, see [Create Space document](../9.space-statements.md/1.create-space.md).
+
+<!-- The `VID` must be unique in the graph space. The current sorting basis is "binary coding order", i.e. 0, 1, 2, ... 9223372036854775807, -9223372036854775808, -9223372036854775807, ..., -1. `VID` supports specifying ID manually, or call hash() function to generate. -->
+
+* `prop_value_list` must provide the property values according to the `prop_name_list`. If the property values do not match the data type in the tag, an error is returned. When the `NOT NULL` constraint is set for a given property, an error is returned if no property is given. When the default value for a property is `NULL`, you can omit to specify the property value. (TODO: create tag doc)
+
+`prop_value_list` must provide the values in the sequence of the properties given in `prop_name_list`. If a property value does not match the data type specified in the tag, an error is returned. When the `NOT NULL` constraint is set for a given property, an error occurs if no value is given. When the default value for a property is `NULL`, you can leave it blank and no error occurs.
```

Comment:
maybe we should mention that some property values have a length limit which is specified in CREATE TAG.

Reply:
TODO

Reply:
but I have an example here